_config.yml

```diff
@@ -32,33 +32,33 @@ exclude:
     "_docs/cleanup.sh",
   ]
 navbar_items:
-  - name: "Assignments"
-    link: "/assignments.html"
+  - name: "Staff"
+    link: "/staff.html"
     sublinks:
-      - name: "Grade Viewer"
-        link: "https://sp19-cs241-grd-02.cs.illinois.edu/grades"
-      - name: "Broadway On-Demand Autograder"
-        link: "https://sp19-cs241-grd-02.cs.illinois.edu/on-demand/"
-  - name: "Quizzes"
-    link: "/quiz_topics.html"
-  - name: "Help!"
-    link: "/help.html"
+        - name: "Staff Hall of Fame"
+          link: "/staff_hall_of_fame.html"
+  - name: "Syllabus"
+    link: "/syllabus.html"
   - name: "Schedule"
     link: "/schedule.html"
     sublinks:
       - name: "Lectures"
         link: "https://classtranscribe.illinois.edu/"
+  - name: "Assignments"
+    link: "/assignments.html"
+  - name: "Quizzes"
+    link: "/quiz_topics.html"
+  - name: "Grades"
+    link: "https://sp19-cs241-grd-02.cs.illinois.edu/grades"
```

Comment:
this link isn't permanent :( maybe we should have another page/link on this website that just redirects to whatever link is currently hosting the grade viewer? Or maybe we should acquire another domain name for the grade viewer?

This isn't something that needs to be fixed in this PR, but maybe something to keep track of

Reply:
Good idea. I will make some more site-wide constants for some of these things.